### ISSUE TITLE
Dataset download issue

### ISSUE THREAD

**Issue by aha097** [prateekchhikara/FIRE#1]
Hello, 
I can't download dataset by following your readme
First, there's no download.py only download_data.py
Also, there's no dataset.json only dataset_sample.json.
Is just incomplete or there's another version.
Sincerely

**Comment by prateekchhikara** [comment]
Hi, the dataset has been removed recently from the official website. Hence, the json files are not publicly available. You can reach out to the authors of Recipe1M dataset.

**Comment by aha097** [comment]
😵‍💫😵‍💫oh my god, I have sent several letters to the authors of Recipe1M dataset but haven't recieved any reply. Do you know any alternative download site🙏🙏? I can't find it with google search.

**Comment by prateekchhikara** [comment]
@aha097 I do not think the dataset is available anywhere. I tried finding it too.

**Comment by jH144270429** [comment]
Hello, have you found this dataset now? It should have been open source before, and it seems that this dataset was deleted recently.

**Comment by JiamingLiu5** [comment]
That's too bad, they deleted it. I can't find the dataset in other open source too, so I try to recollect one, it's very time-consuming.

**Comment by jH144270429** [comment]
Yes, I am looking for this dataset too. I want ingredients, images, and recipes. The food segmentation dataset I found only has ingredients, and another dataset only has ingredients and recipes.

> 太糟糕了，他们把它删掉了。我在其他开源中也找不到这个数据集，所以我试着重新找一个，这很耗时。

**Comment by prateekchhikara** [comment]
Hello @aha097 @AndrewAha @jH144270429, I was finally able to get the dataset and I have uploaded it on drive. Hope this helps.

https://drive.google.com/drive/folders/16r2rshK9CKEdAXZFlz6IeJq5uzejwyJC?usp=sharing

**Comment by zzYssii** [comment]
Hello,
I found there are 3 json files through the link but I dont know what these 3 json files refer to. If dataset.json includes all these 3 files?
And, there's no download.py only download_data.py. Are them the same?
